cli/things.go

```diff
@@ -40,21 +51,24 @@ var cmdThings = []cobra.Command{
 				logUsage(cmd.Short)
 				return
 			}
+
 			if args[0] == "all" {
-				l, err := sdk.Things(args[1])
+				l, err := sdk.Things(args[1], uint64(Offset), uint64(Limit))
 				if err != nil {
 					logError(err)
 					return
 				}
-				dump(l)
+				flush(l)
 				return
 			}
+
 			t, err := sdk.Thing(args[0], args[1])
 			if err != nil {
 				logError(err)
 				return
 			}
-			dump(t)
+
+			flush(t)
```

Comment:
`flush` != `dump` - flush flushes the stdio buffer while `dump` just dumps/prints the stuff on screen. You can change `dump`, but make it something else than `flush`.

Reply:
I know it's different, but `dump` sounds... how should I say this... wrong... Also, this CLI should be refactored anyway, so maybe this naming can be fixed in that PR?